tests/lib/devicetree/devices/src/main.c

```diff
@@ -58,7 +58,7 @@ DEVICE_DEFINE(manual_dev, "Manual Device", dev_init, NULL,
 #define DEV_HDL(node_id) device_handle_get(DEVICE_DT_GET(node_id))
 #define DEV_HDL_NAME(name) device_handle_get(DEVICE_GET(name))
 
-static void test_init_get(void)
+ZTEST(devicetree_driver, test_init_get)
```

Comment:
Can you please use `devicetree_devices` instead of `devicetree_driver`? Using "devices" matches the file system path.

Reply:
> Can you please use `devicetree_devices` instead of `devicetree_driver`? Using "devices" matches the file system path.

ok,I have done